drbg/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "tpm2-rs-drbg"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+crypto-bigint = {workspace = true, features = ["generic-array"]}
+digest = { workspace = true }
+sha2 = {workspace = true, optional = true}
+tpm2-rs-server = { workspace = true }
+
+[features]
+# Implement HashDrbgProps for hash functions from sha2
+rustcrypto = ["dep:sha2"]
```

Comment:
Light request for a newline between sections (e.g. a newline after the `rustcrypto = ["dep:sha2"]` and before `[dev-dependencies]`)